language/tools/read-write-set/types/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
-name = "read-write-set-types"
-version = "0.1.0"
+name = "move-read-write-set-types"
+version = "0.0.1"
```

Comment:
You'll need to move the `diem-workspace-hack` dependency to a `[dev-dependency]` section and then change the `publish` field to `[crates.io]`

Reply:
You'll also be required to do exactly the same for the `move-binary-format` crates because that one will also be required to be published for this one to be published. Its a slippery slope ;)

Reply:
Fixed!